pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/MessageDeduplication.java

```diff
@@ -139,6 +141,7 @@ public MessageDeduplication(PulsarService pulsar, PersistentTopic topic, Managed
         this.maxNumberOfProducers = pulsar.getConfiguration().getBrokerDeduplicationMaxNumberOfProducers();
         this.snapshotCounter = 0;
         this.replicatorPrefix = pulsar.getConfiguration().getReplicatorPrefix();
+        this.chunkMessageOngoing = ConcurrentOpenHashMap.<String, Long>newBuilder().build();
```

Comment:
What if a broker restarts in the middle of receiving chunked messages?

Reply:
I feel like we need a community discussion on this problem.

Reply:
Fix